pkg/sdkserver/sdkserver.go

```diff
@@ -547,21 +555,68 @@ func (s *SDKServer) stopReserveTimer() {
 // [Stage:Alpha]
 // [FeatureFlag:PlayerTesting]
 func (s *SDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
-	panic("implement me")
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	s.logger.WithField("PlayerId", id).Debug("Player Connected")
+
+	s.gsUpdateMutex.RLock()
+	// check if the player id is in the player map
+	_, ok := s.connectedPlayers[id.PlayerId]
+	s.gsUpdateMutex.RUnlock()
+
+	// this player has already connected, so exit
+	if ok {
+		return &alpha.Empty{}, nil
```

Comment:
Should this case return an error ("player already registered")? Right now there are no error returns unless the feature flag is disabled.

Reply:
Since this going back up to gRPC, rather than throwing an error (which we could pick up if this was just Go), should PlayerConnect and PlayerDisconnect return a bool value, as to whether or not they were successful?